### DISCUSSION TITLE
Run two commands together with listening variables

### DISCUSSION THREAD

**Discussion by wworeno** [elkowar/eww#1100] [Q&A]
I'm setting up a bar using eww and a question arose when using the listening variable. In my code I have the following line:

`(deflisten net-stat :initial "n/a" 'tail -F my-bar/widgets/right-widgets/vars-listened/net-stat | awk -F: "{print \$1}"')`

Which repeatedly read the "net-stat" file which contains a single line "connected:full:enabled".

The objective is to pass the first content of the line (connected) to the variable, but in practice the value is not passed and the variable continues with its initial value "n/a".

When executing the same command in the terminal it works as expected. I can't understand why, because when trying other commands together, it works normally. Thank you to anyone who can shed some light on this.

**Comment by MatthieuBarthel** [comment]
Hi, I think you need a line break after the content of `net-stat` file
